src/App.js

```diff
@@ -11,29 +11,47 @@ import getloc from './utils/getloc';
 import getdate from './utils/getdate';
 
 
-/*eslint-disable */
-const mapStyle = [{'featureType':'administrative.neighborhood','elementType':'labels.text','stylers':[{'visibility':'simplified'}]},{'featureType':'all','elementType':'labels.text.fill','stylers':[{'color':'#ffffff'}]},{'featureType':'all','elementType':'labels.text.stroke','stylers':[{'color':'#000000'},{'lightness':13}]},{'featureType':'administrative','elementType':'geometry.fill','stylers':[{'visibility':'off'},{'color':'#000000'}]},{'featureType':'administrative','elementType':'geometry.stroke','stylers':[{'color':'#144b53'},{'weight':1.4},{'lightness':14}]},{'featureType':'landscape','elementType':'all','stylers':[{'color':'#08304b'}]},{'featureType':'road.highway','elementType':'geometry.fill','stylers':[{'visibility':'on'},{'color':'#000000'}]},{'featureType':'road.highway','elementType':'geometry.stroke','stylers':[{'visibility':'on'},{'color':'#0b434f'},{'lightness':25}]},{'featureType':'road.arterial','elementType':'geometry.fill','stylers':[{'color':'#000000'}]},{'featureType':'road.arterial','elementType':'geometry.stroke','stylers':[{'color':'#0b3d51'},{'lightness':16}]},{'featureType':'road.local','elementType':'geometry','stylers':[{'color':'#000000'}]},{'featureType':'transit','elementType':'all','stylers':[{'visibility':'off'},{'color':'#146474'}]},{'featureType':'water','elementType':'all','stylers':[{'color':'#021019'}]},{'featureType':'poi','elementType':'all','stylers':[{'visibility':'off'}]},{'featureType':'road.highway','elementType':'labels.icon','stylers':[{'visibility':'off'}]},{'featureType':'road.arterial','elementType':'labels.text','stylers':[{'visibility':'off'}]},{'featureType':'road.local','elementType':'labels','stylers':[{'visibility':'off'}]},{'featureType':'administrative.land_parcel','elementType':'all','stylers':[{'visibility':'on'}]},{'featureType':'administrative.locality','elementType':'all','stylers':[{'visibility':'off'}]},{'featureType':'road','elementType':'labels.icon','stylers':[{'visibility':'off'}]}];
-/*eslint-enable */
-
 export class App extends Component {
   constructor(props) {
     super(props);
     this.state = {
       video: { id: { videoId: '' }, snippet: { title: '' } },
-      map: null,
       events: [],
-      mapStyle: mapStyle, //eslint-disable-line
+      currentEvent: {},
       fail: false,
+      startDate: getdate('yyyy-mm-dd'),
+      endDate: getdate('yyyy-mm-dd'),
     };
-  }
 
-  componentDidMount() {
-    // gets the date from utils/getdate
-    const today = getdate('yyyy-mm-dd');
-    // gets loc from utils/getloc
     getloc((currlocation) => {
-      // call getQuery on loc and date
-      this.getQuery(currlocation, today, today, 'music');
+      this.getQuery(currlocation, this.state.startDate, this.state.endDate, 'music');
+    });
+  }
+
+  changeLatLng(lat, lng) {
+    this.setState({
+      lat,
+      lng,
```

Comment:
Zack said that even though we can create new states throughout the project, we should instantiate them in the constructor with a null or empty string so there isn't confusion.